src/components/Search/Search.tsx

```diff
@@ -10,10 +10,22 @@ import Meta from '../Meta'
 import { canSearchForProperty } from '@/utils/userPermissions'
 import { PropertyListItem } from '@/models/propertyListItem'
 
-const Search = ({ query }) => {
+interface SearchProps {
+  query?: {
+    searchText?: string
+    pageNumber?: string
```

Comment:
Seems like it was decided to go with the `pageNumber` as a string rather than a number. So long as everyone is happy about it, I can approve.